CONTRIBUTING.md

```diff
@@ -0,0 +1,9 @@
+# Contributing
+
+If you would like to make changes to this repo, please:
+
+1. Make a GitHub Pull Request with the suggested change
+2. Post the Pull Request in the Slack channel: #Frontend
+3. Wait a day or two (at least 24 hours) for feedback
+4. Ping the #Frontend channel again
+5. If you have at least a few approvals, and no one is hard disapproving of your PR, merge it
```

Comment:
Should consider expanding in the case that someone is "hard disapproving": Something like:

> If it’s contentious and cannot reach agreement, the Frontend Platform team will discuss and make a stance to avoid everyone swirling.

Perhaps that's too heavy-handed, but might help move things reach resolution quicker.